plugins/reach/services/EntryVendorTaskService.php

```diff
@@ -460,10 +460,11 @@ public function extendAccessKeyAction($id)
 		
 		$shouldModerateOutput = $dbEntryVendorTask->getIsOutputModerated();
 		$accessKeyExpiry = $dbEntryVendorTask->getAccessKeyExpiry();
+		$dbVendorCatalogItem = VendorCatalogItemPeer::retrieveByPK($dbEntryVendorTask->getCatalogItemId());
 		
```

Comment:
better check that $dbVendorCatalogItem actually returned like you did in kReachManager

Reply:
but in this case if its null it will get exception on the try the same thing we will do it if we will check it so it the same flow no ?